jbosstools/mars/staging/updates/compositeArtifacts.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <?compositeArtifactRepository version='1.0.0'?>
-<repository name='JBoss Tools 4.3.1.Beta2a Development Milestone Update Site' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
+<repository name='JBoss Tools 4.3.0.Final Stable Release Update Site (With EGit 4.1.1)' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
 <properties size='2'>
 <property name='p2.compressed' value='true'/>
-<property name='p2.timestamp' value='1453955143000'/>
+<property name='p2.timestamp' value='1457972311000'/>
 </properties>
-<children size='5'>
-<child location='http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.52.0.Beta2-SNAPSHOT/REPO/'/>
-<child location='http://download.jboss.org/jbosstools/mars/staging/updates/core/4.3.1.Beta2a/'/>
-<child location='http://download.jboss.org/jbosstools/targetplatforms/jbtcentraltarget/4.52.0.Beta2-SNAPSHOT/REPO/'/>
-<child location='http://download.jboss.org/jbosstools/mars/staging/updates/central/4.3.1.Beta2a/'/>
-<child location='http://download.jboss.org/jbosstools/mars/staging/updates/integration-stack/'/>
+<children size='6'>
+<child location='http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.51.0.Final/REPO/'/>
```

Comment:
I can't find my original comment but I believe I said next time to do it right and do it *under* /staging/ does not mean to go back in versions on staging - would have made more sense to have had a /staging/jbide-21281 folder for this one off thing so we would not back in versions.

And I'm not talking about going to 4.3.1.Beta2 specifically, I'm pointing out it is weird our mars staging site is going backwards in content. Thus if 4.3.1.CR1 is the right one to have then that is of course better.